pretix_newsletter_manual/exporters.py

```diff
@@ -44,11 +45,15 @@ def iterate_list(self, form_data):
                 r.order.email,
                 r.created.astimezone(tz).strftime("%Y-%m-%d"),
             ]
-            if r.order.invoice_address and r.order.invoice_address.name:
-                row += [r.order.invoice_address.name]
+            try:
+                invoice_address = r.order.invoice_address
+            except InvoiceAddress.DoesNotExist:
+                invoice_address = False
```

Comment:
As invoice_address is expected to be an object, I would find it semantically better to default it to `None` instead of `False`. Practically it does not make a difference *in this case.*

```suggestion
                invoice_address = None
```